guides/common/modules/ref_host-parameters-in-ansible-playbooks.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="host-parameters-in-ansible-playbooks"]
+= Host parameters in Ansible Playbooks
+
+You can use host parameters from {Project} in Ansible Playbooks.
+
+.Example Ansible Playbook
+[source, yaml, options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+---
+- name: "Ansible Playbook to access host parameters"
+  hosts: all
+  gather_facts: true
+  tasks:
+    - name: "Use a host parameter"
+      ansible.builtin.debug:
+        var: hostvars[inventory_hostname]["_My_Host_Parameter_"]
+
+    - name: "Use ERB templating"
+      ansible.builtin.debug:
+        # value of _My_Host_Parameter_: "<%= @host.capabilities.include?(:snapshots) %>"
```

Comment:
How does `<%= @host.capabilities.include?(:snapshots) %>` resolve to `value of _My_Host_Parameter_`?